Dockerfile

```diff
@@ -7,10 +7,16 @@ ARG BASE_IMAGE="ghcr.io/mit-dci/opencbdc-tx-base:latest"
 # Create Base Image
 FROM $IMAGE_VERSION AS base
 
+# make shell run with bash not sh
+SHELL ["/bin/bash", "-c"]
+
 # set non-interactive shell
-ENV DEBIAN_FRONTEND noninteractive
-ENV CMAKE_BUILD_TYPE Release
-ENV BUILD_RELEASE 1
+ENV DEBIAN_FRONTEND=noninteractive
+ENV CMAKE_BUILD_TYPE=Release
+ENV BUILD_RELEASE=1
```

Comment:
Fix for these legacy warnings 
```console
3 warnings found (use --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 11)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 12)
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 13)
```